Advice

robertoninob opened the pull request:
I added Roberto Nino’s Advice.

redrambles commented:
Hello @robertoninob !

Congrats on your pull request!! Thanks for making those little changes in another student's link - and we are nearly good to go here, except that when you added your link, you overwrote the link of another student, see here:


![CleanShot 2023-11-10 at 09 48 34](https://github.com/skillcrush/git-practice/assets/4411121/d8481615-628f-4ec1-bcca-06131d5e6cc4)

No worries! All we need to do is copy/paste Cassandra's link from the [previous changeset](https://github.com/skillcrush/git-practice/pull/1176/files) and make sure to include it above your link, so that both of you are on the page. Once you have done that, add and commit the change and push up the `advice` branch once more, which will automatically update this pull request.

Let us know if you have any questions!

Cheers,
Ann :)

redrambles commented:
Perfect!

Your code is merged in, @robertoninob 🎉

Cheers,
Ann :)

robertoninob commented:
Hi @redrambles, I fixed the index.html and added Cassandra's comment. However, as I was about to merge I was notified of a conflict. Adra Conde's commet was added in the meantime apparently. I just merged the modified index.html with all of our comments, everything seems fine now 🌟
